### ISSUE TITLE
Slathering with multiple binary_basenames dont merge results in cobertura.xml

### ISSUE THREAD

**Issue by jnes92** [SlatherOrg/slather#444]
Hey boys and girls,

im running slather with the fastlane action for slather and want to produce code coverage results from multiple binaries.

my slather config looks like this:
```
slather(
        ....
        cobertura_xml: true,

        binary_basename: "App",
        binary_basename: "Module1",
        binary_basename: "Module2"
```

But I noticed that the resulting XML file just contains information for files from the last package, so 'Module2'.
Im not sure, if its only accuring for cobertura_xml ?

**Comment by weibel** [comment]
I don't think you can put in multiple keys with the same name i.e. `binary_basename`. One of them will overwrite the others. If it should work it should be an array 
`binary_basename: ["App", "Module"]` but it does not look like the fastlane action supports this.

Before you open an issue with Fastlane read https://github.com/SlatherOrg/slather/issues/440 to find out if Slather can do what you are trying to accomplish.

**Comment by jnes92** [comment]
I was also surprised during the time I found out:

https://github.com/SlatherOrg/slather/pull/188

See: Example usage:

slather coverage --scheme MyScheme --binary-basename BinaryOne --binary-basename DynamicLibrary -s MyProject.xcodeproj

I will try it again, if I can use that parameter as an array inside fastlane and give an update here :)
